src/makielayout/lineaxis.jl

```diff
@@ -575,6 +575,11 @@ function get_ticks(::Automatic, scale::Union{typeof(log10), typeof(log2), typeof
     get_ticks(LogTicks(WilkinsonTicks(5, k_min = 3)), scale, any_formatter, vmin, vmax)
 end
 
+# the hyphen which is usually used to store negative number strings
+# is shorter than the dedicated minus in most fonts, the minus glyph
+# looks more balanced with numbers, especially in superscripts or subscripts
+const MINUS_SIGN = "−"
```

Comment:
I actually think the hyphen looks more pleasing in the screenshots... But not a hill to die on for me ;)